pkg/datagatherer/k8s/fieldfilter.go

```diff
@@ -25,6 +25,26 @@ var SecretSelectedFields = []string{
 	"/data/ca.crt",
 }
 
+// RouteSelectedFields is the list of fields sent from OpenShift Route objects to the
+// backend
+var RouteSelectedFields = []string{
+	"kind",
+	"apiVersion",
+	"metadata.annotations",
+	"metadata.name",
+	"metadata.namespace",
+	"metadata.ownerReferences",
+	"metadata.selfLink",
+	"metadata.uid",
+	"spec.host",
+	"spec.to.kind",
+	"spec.to.name",
+	"spec.tls.termination",
+	"spec.tls.certificate",
+	"spec.tls.caCertificate",
+	"spec.tls.destinationCACertificate",
```

Comment:
From the [docs](https://docs.openshift.com/container-platform/3.11/rest_api/route_openshift_io/route-route-openshift-io-v1.html), I only see potential secret info in `spec.tls`.

```suggestion
	"kind",
	"apiVersion",
	"metadata.annotations",
	"metadata.name",
	"metadata.namespace",
	"metadata.ownerReferences",
	"metadata.selfLink",
	"metadata.uid",
	"spec.host",
	"spec.to.kind",
	"spec.to.port",
	"spec.to.name",
	"spec.to.weight",
	"spec.tls.termination",
	"spec.tls.certificate",
	"spec.tls.caCertificate",
	"spec.tls.destinationCACertificate",
	"spec.tls.insecureEdgeTerminationPolicy",
	"spec.wildcardPolicy",
	"status",
```

Reply:
Thank you @j-fuentes that looks good!